src/geoaxis.jl

```diff
@@ -104,3 +102,27 @@ function GeoAxis(args...;
     hidespines && hidespines!(ax)
     return ax
 end
+
+
+function Makie.apply_transform(f::Proj4.Transformation, pt::Point)
+    Point(f(Vec(pt)) ./ 10_000f0)
+end
+function Makie.apply_transform(f::Proj4.Transformation, r::Rect2)
+    # TODO: once Proj4.jl is updated to PROJ 8.2, we can use
```

Comment:
Ah good catch. Another reason to finish https://github.com/JuliaGeo/Proj4.jl/pull/57.